### DISCUSSION TITLE
Updatable edge always selecting the last added edge if there are multiple edges from the same handle

### DISCUSSION THREAD

**Discussion by kaushalendra-pandey** [xyflow/xyflow#2557] [Q&A]
I have multiple edges coming out from the same handle. When I try to drag one these edges from on handle to different only the last added edges is getting selected. For example , in the image every-time the red edge will get selected for update as it was last added.

How can I make sure that user can update any edge coming from the handle?

<img width="131" alt="image" src="https://user-images.githubusercontent.com/67643722/201862903-0c2fe3fc-b7f3-4b42-b050-2db21d132433.png">

**Comment by coolRoger** [comment]
@moklick I need help on this too

**Comment by coolRoger** [comment]
finally come out with a workaround:

```
const onEdgeClick: ReactFlowProps["onEdgeClick"] = useMemoizedFn(
        (event, node) => {
            console.log(node);
            let new_edges:Edge[] = edges.map((edge:Edge) => {
                if(edge.id === node.id){
                    edge.zIndex = 999;
                    edge.selected = true;
                }else{
                    edge.zIndex = 1;
                }

                return edge;
            });

            setEdges(new_edges);
        }
    );
```

set zIndex when the edge is clicked, and updater handle will on the top of all edges.
